website/docs/developer/research/research-reconciliation.mdx

```diff
@@ -661,71 +660,100 @@ shown below.
     resetPic={useBaseUrl('img/zoom-pan-reset.svg')}
 />
 
+#### Status Table reconcile_status
+
+Refrence table for valid status values and status order.
+
+| Attribute Name | Data Type | Description                  | Notable Items   |
+| -------------- | --------- | ---------------------------- | --------------- |
+| id             | `int2`    | Status ID.                   | Primary Key     |
+| value          | `string`  | Human readable status value. |  |
+
 #### Internal Reconciliation Jobs Table reconcile_jobs
 
 This table would capture internal reconciliation job information. Below is a
 sampling of attributes that may be needed. Based on design and implementation,
 additional attributes may be required. This table, in particular may need additional
 items for managing jobs.
 
-| Attribute Name | Data Type | Description                               | Notable Items   |
-| -------------- | --------- | ----------------------------------------- | --------------- |
-| id             | `int8`    | Unique Job ID.                            | Primary Key     |
-| status         | `string`  | Current status of the job. An Enumerated string. | Generally reflect the stages of reconciliation. |
-| start_time     | `timestamptz` | Start time of the job.                | Cannot be NULL  |
-| end_time       | `timestamptz` | End time of the job.                  | NULLABLE column |
-| last_update    | `timestamptz` | Last time the status was updated.     | Cannot be NULL  |
-| error_message  | `string`  | Job error message in cases of failure.    |NULLABLE column  |
+| Attribute Name          | Data Type    | Description                                 | Notable Items   |
+| ----------------------- | ------------ | ------------------------------------------- | --------------- |
+| id                      | `int2`       | Unique Job ID.                              | Primary Key     |
+| orca_archive_location   | `string`     | ORCA S3 Glacier bucket the file resides in. | Location in S3. |
+| status_id               | `int8`       | Current status of the job.                  | Generally reflect the stages of reconciliation. |  TODO: Switch to int/string pairs
+| inventory_creation_time | `timestampz` | Time from the s3 inventory manifest         | Cannot be NULL  |
+| start_time              | `timestamptz`| Start time of the job.                      | Cannot be NULL  |
+| last_update             | `timestamptz`| Last time the status was updated.           | Cannot be NULL  |
+| end_time                | `timestamptz`| End time of the job.                        | NULLABLE column |
+| error_message           | `string`     | Job error message in cases of failure.      | NULLABLE column |
```

Comment:
currently error_message is not present in `reconcile_jobs` table in the architecture drawing.

Reply:
Fixed a couple issues there.